README.md

```diff
@@ -67,36 +67,17 @@ Currently we are manually building an archive in xCode to distribute the app.
 4. Select Product > Archive to create the app archive.
 5. If your archive builds correctly it will walk you through pushing this to the app store from there. 
 
-### Running on device with local web
-1. Run web on your local machine as usual
-
-2. install ngrok
-```
-brew install ngrok
-```
-1. Create a free account here https://ngrok.com/ and obtain your auth token
-4. Configure ngrok with your token
-```
-ngrok config add-authtoken YOUR_TOKEN
-```
-3. run ngrok, note the public `Forwarding` domain listed
-```
-ngrok http 3000 # substitute port if necessary
-```
-4. Modify `.env` to point the WebView at the ngrok endpoint created above
-```
-SHAPESHIFT_URI=https://523d-135-134-162-133.ngrok.io # example ngrok domain
-```
-5. Run the app on your device from Xcode _or_ use the yarn script:
+### Running on Expo Go with local web
+- Run web, select the lan IP
```

Comment:
```suggestion
- Run web, copy the lan IP e.g `192.168.x.y`
```

Reply:
Updated in some commits I don't remember